shared-libs/search/src/generate-search-requests.js

```diff
@@ -10,13 +10,12 @@ var getKeysArray = function(keys) {
 
 // filter = { selected: [...], options: [...]}
 var getRequestForMultidropdown = function(view, filter, mapKeysFunc) {
-  if (!filter || !filter.selected || !filter.options) {
+  if (!filter || !filter.selected) {
     return;
   }
 
-  // If everything is selected, no filter to apply.
-  var everythingSelected = filter.selected.length === filter.options.length;
-  if (everythingSelected) {
+  // If we know everything is selected, no filter to apply.
+  if (filter.options && filter.selected.length === filter.options.length) {
```

Comment:
This and above is getting rid of the need to pass `filter.options` if you don't want to.